pyproject.toml

```
@@ -267,7 +267,7 @@ markers = [
 # - MuslLinux (tests hang non-deterministically)
 test-skip = "*-musllinux_x86_64"
 enable = ["cpython-prerelease"] # this line can be removed when cibuildwheel is bumped to 3.1.0 or newer
-skip = ["cp314t-*"] # https://github.com/astropy/astropy/issues/16916
+skip = ["cp31?t-*"] # https://github.com/astropy/astropy/issues/16916
```

Even more future proof !

```suggestion
skip = ["cp3*t-*"] # https://github.com/astropy/astropy/issues/16916
```